index.html

```diff
@@ -14,6 +15,103 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+      <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+
+        <header>
+          <div class="header-image">
+              <img src="img/aboutimg.JPG" />
+          </div>
+
+          <div class="header-text">
+            <h1>Ezekiel Ekunola</h1>
+            <h3>Software Engineer</h3>
+            <p>Experienced Web Developer with a demonstrated history of working in the information technology and services industry. Skilled in NODE, REACT, MONGODB, PHP, JAVASCRIPT, HTML, Search Engine Optimization (SEO), and Computer Hardware. Strong engineering professional graduated from Aptech Computer Education.</p>
+          </div>
+        </header>
+
+        <section class="skills">
+          <h2>Skills</h2>
+
+          <button class="skill-button green"><p class="skill-button-text"><i class="fab fa-node"></i>  Node</p></button>
+          <button class="skill-button blue"><p class="skill-button-text"><i class="fab fa-react"></i>  React</p></button>
+          <button class="skill-button purple"><p class="skill-button-text">Redux</p></button>
+          <button class="skill-button green"><p class="skill-button-text">Mongodb</p></button>
+          <button class="skill-button blue"><p class="skill-button-text"><i class="fab fa-php"></i> Php</p></button>
+          <button class="skill-button red"><p class="skill-button-text"><i class="fab fa-laravel"></i> Laravel</p></button>
+          <button class="skill-button red"><p class="skill-button-text">Codeigniter</p></button>
+          <button class="skill-button blue"><p class="skill-button-text">Yii</p></button>
+          <button class="skill-button blue"><p class="skill-button-text"><i class="fab fa-java"></i> Java</p></button>
+          <button class="skill-button yellow"><p class="skill-button-text"><i class="fab fa-js"></i> Javascript</p></button>
+          <button class="skill-button black"><p class="skill-button-text"><i class="fab fa-github"></i> Github</p></button>
+          <button class="skill-button red"><p class="skill-button-text"><i class="fab fa-gitlab"></i> Gitlab</p></button>
+          <button class="skill-button blue"><p class="skill-button-text"><i class="fab fa-wordpress"></i> Wordpress</p></button>
+          <button class="skill-button black"><p class="skill-button-text"><i class="fab fa-joomla"></i> Joomla</p></button>
+          <button class="skill-button purple"><p class="skill-button-text">Heroku</p></button>
+          <button class="skill-button orange"><p class="skill-button-text"><i class="fab fa-aws"></i> Aws</p></button>
```

Comment:
I don't really love the use of buttons here. They are intended to be clicked and to just to renders some text on the page.
Also I think you could try to refactor your code to don't have all those different classes for just few characters of text.

It's working and it's fine, but I feel it could be cleaner.